File: templates/Uwp/_composition/_shared/Page.Ink.AddBasicInkServices/EventHandlers/Ink/AddStrokeEventArgs.cs
using System;
using Windows.UI.Input.Inking;

namespace Param_ItemNamespace.EventHandlers.Ink
{
    public class AddStrokeEventArgs : EventArgs
    {
        public InkStroke OldStroke { get; set; }

        public InkStroke NewStroke { get; set; }

        public AddStrokeEventArgs(InkStroke newStroke, InkStroke oldStroke = null)
        {
            NewStroke = newStroke;
            OldStroke = oldStroke;
        }
    }
}

File: code/TemplateStudioForUWP/Templates/Uwp/_comp/_shared/Page.Ink.AddBasicInkServices._VB/EventHandlers/Ink/AddStrokeEventArgs.vb
Imports Windows.UI.Input.Inking

Namespace EventHandlers.Ink
    Public Class AddStrokeEventArgs
        Inherits EventArgs

        Public Property OldStroke As InkStroke

        Public Property NewStroke As InkStroke

        Public Sub New(newStroke As InkStroke, Optional oldStroke As InkStroke = Nothing)
            Me.NewStroke = newStroke
            Me.OldStroke = oldStroke
        End Sub
    End Class
End Namespace
